Makefile

```diff
@@ -27,7 +27,7 @@ LD_FLAGS    ?= -X $(REPO_PATH)/pkg/version.Raw=$(VERSION) -extldflags -static
 BUILD_IMAGE ?= registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-openshift-4.18
 
```

Comment:
The `test` target's  ENVTEST invocation will need to change to something like https://github.com/openshift/machine-api-operator/blob/2f48e767b57217c37921e090b34b73c997d886a9/Makefile#L106

Now that you bumped the ENVTEST_K8S_VERSION to 1.32.1

Reply:
Do you mean how I've changed it to now? Thanks!